src/components/TaxServiceModal/TaxServiceBanner.tsx

```diff
@@ -0,0 +1,165 @@
+  box-shadow: ${({ theme }) => theme.deepShadow};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  background-color: ${({ theme }) => theme.backgroundSurface};
+  border-radius: 13px;
+  cursor: pointer;
+  color: ${({ theme }) => theme.textPrimary};
+  display: ${({ show }) => (show ? 'flex' : 'none')};
+  flex-direction: column;
+  position: fixed;
+  right: clamp(0px, 1vw, 16px);
+  z-index: ${Z_INDEX.sticky};
+  transition: ${({
+    theme: {
+      transition: { duration, timing },
+    },
+  }) => `${duration.slow} opacity ${timing.in}`};
+  width: 320px;
+  height: 156px;
+  @media screen and (min-width: ${SMALLEST_MOBILE_MEDIA_BREAKPOINT}) {
+    bottom: 50px;
+  }
+
+  ::before {
+    content: '';
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    bottom: 0px;
+    left: 0px;
+
+    background-image: url(${CointrackerLogo}), url(${TokenTaxLogo});
+    background-size: 15%, 20%;
+    background-repeat: no-repeat;
+    background-position: top right 75px, bottom 5px right 7px;
+
+    opacity: ${({ isDarkMode }) => (isDarkMode ? '0.9' : '0.25')};
+  }
+`
+
+const InnerContainer = styled.div`
+  border-radius: 12px;
+  cursor: auto;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  gap: 8px;
+  padding: 16px;
+  ::before {
+    content: '';
+    position: absolute;
+    top: 0px;
+    right: 0px;
+    bottom: 0px;
+    left: 0px;
+
+    background-color: ${({ theme }) => theme.accentActionSoft};
+    opacity: 0.4;
+  }
+`
+
+const Button = styled(ThemeButton)`
+  margin-top: auto;
+  margin-right: auto;
+  padding: 8px 24px;
+  gap: 8px;
+  border-radius: 12px;
+`
+
+const TextContainer = styled.div`
+  user-select: none;
+  display: flex;
+  flex-direction: column;
+  width: 70%;
+  justify-content: center;
+`
+
+export const StyledXButton = styled(X)`
+  color: ${({ theme }) => theme.textPrimary};
+  cursor: pointer;
+  &:hover {
+    opacity: ${({ theme }) => theme.opacity.hover};
+  }
+  &:active {
+    opacity: ${({ theme }) => theme.opacity.click};
+  }
+`
+
+const TAX_SERVICE_DISMISSED = 'TaxServiceToast-dismissed'
+
+export default function TaxServiceBanner() {
+  const isDarkMode = useIsDarkMode()
+  const [modalOpen, setModalOpen] = useState(false)
+  const sessionStorageTaxServiceDismissed = sessionStorage.getItem(TAX_SERVICE_DISMISSED)
+  let initialBannerOpen = true
+  if (!sessionStorageTaxServiceDismissed) {
+    sessionStorage.setItem(TAX_SERVICE_DISMISSED, 'false')
+  } else if (sessionStorageTaxServiceDismissed === 'true') {
+    initialBannerOpen = false
+  }
+  const [bannerOpen, setBannerOpen] = useState(initialBannerOpen)
+  const onDismiss = () => {
+    setModalOpen(false)
+  }
+
+  const toggleTaxModal = () => {
+    setModalOpen(true)
+  }
+
+  const handleClose = useCallback(() => {
+    setBannerOpen(false)
+    sessionStorage.setItem(TAX_SERVICE_DISMISSED, 'true')
+  }, [])
```

Comment:
this worked the first time I clicked it, but when I closed the tab and returned, the dismissal no longer functioned and I didn't get a `cursor: pointer`

![image](https://user-images.githubusercontent.com/5773490/223858220-4a302e4c-8d23-4dc4-834c-a7fec6fc6c74.png)

Reply:
fixed thanks for catching, it was the pseudo ::before element I used for styling on InnerContainer that was interfering. implemented design requested changes in a diff way. behaves as expected now.